### PR TITLE
Optimize/parse kubefile log and fix umount /tmp/cluster Dir

### PR DESCRIPTION
**Pull request by bxy4543**
### Describe what this PR does / why we need it
Optimize/parse kubefile log and fix umount /tmp/cluster Dir

### CODE REVIEW

File: apply/cloud_applier.go [hunk 1]
```diff
@@ -56,7 +56,11 @@ func (c *CloudApplier) Apply() error {
 	if err != nil {
 		return fmt.Errorf("marshal tmp cluster file failed %v", err)
 	}
-
+	defer func() {
+		if err := utils.CleanFiles(common.TmpClusterfile); err != nil {
+			logger.Error(fmt.Errorf("failed to clean %s,err:%v", common.TmpClusterfile, err))
```
Comment (justadogistaken) [inline apply/cloud_applier.go:61]: %s, err: %v

File: guest/guest.go [hunk 2]
```diff
@@ -40,6 +44,10 @@ func (d *Default) Apply(cluster *v1.Cluster) error {
 			return err
 		}
 	}
+	mountClusterDir := filepath.Join(os.TempDir(), cluster.Name)
+	if err := mount.NewMountDriver().Unmount(mountClusterDir); err != nil {
+		logger.Error("failed to umount %s,err: &v", mountClusterDir, err)
```
Comment (justadogistaken) [inline guest/guest.go:49]: %s, err: %v

File: parser/parser.go [hunk 3]
```diff
@@ -34,8 +34,10 @@ func (p *Parser) Parse(kubeFile []byte, name string) *v1.Image {
 	for scanner.Scan() {
 		text := scanner.Text()
 		layerType, layerValue, err := decodeLine(text)
-		if err != nil || layerType == "" {
-			logger.Warn("decode kubeFile line failed %v", err)
+		if layerType == "" {
+			if err != nil {
+				logger.Warn("decode kubeFile line failed %v", err)
```
Comment (justadogistaken) [inline parser/parser.go:39]: %s, err: %v

File: apply/cloud_applier.go (hunk 1 above)
Comment (justadogistaken) [reply apply/cloud_applier.go:61]: logger.Error can support format, fmt.Errorf is redundant

**Review by fanux — APPROVED** [review]

**Pull request merged**